install_script.sh.template

```diff
@@ -680,6 +801,9 @@ else
 Please follow the instructions on the Agent setup page:
 
     https://app.datadoghq.com/account/settings#agent\033[0m\n"
+    ERROR_MESSAGE="Unsupported platform"
```

Comment:
Why not use the above `printf` message as the `ERROR_MESSAGE` here, like you did in other places?

Reply:
The message here is quite long and spans multiple lines. I guess I could have used just the first part of the message 🤷‍♂️

Reply:
Actually, some of the other error messages are multiline as well. I updated to code to make it consistent